artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -347,6 +361,27 @@ public String getClusterPassword() {
       return clusterPassword;
    }
 
+   public String getSslKeyPassword() {
+      if (sslKeyPassword == null) {
+         sslKeyPassword = inputPassword("--ssl-key-password", "Please enter the keystore password:", "password");
```

Comment:
I wouldn't force any of this to use input.. this is an extra feature... can you remove the inputs you have added?